rllib/utils/test_utils.py

```diff
@@ -634,7 +634,6 @@ def check_train_results(train_results: PartialAlgorithmConfigDict) -> ResultDict
         "episode_reward_max",
         "episode_reward_mean",
         "episode_reward_min",
-        "episodes_total",
```

Comment:
These two are not populated by rllib anymore. 
We probably should just remove them from Tune as well..